# -*- coding: utf-8 -*-

from PyQt5 import QtCore, QtWidgets

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(508, 409)
        self.centralWidget = QtWidgets.QWidget(MainWindow)
        self.centralWidget.setObjectName("centralWidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralWidget)
        self.gridLayout.setContentsMargins(11, 11, 11, 11)
        self.gridLayout.setSpacing(6)
        self.gridLayout.setObjectName("gridLayout")
        self.quit_button = QtWidgets.QPushButton(self.centralWidget)
        self.quit_button.setMinimumSize(QtCore.QSize(25, 0))
        self.quit_button.setIconSize(QtCore.QSize(64, 16))
        self.quit_button.setObjectName("quit_button")
        self.gridLayout.addWidget(self.quit_button, 1, 1, 1, 1)
        spacerItem = QtWidgets.QSpacerItem(401, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.gridLayout.addItem(spacerItem, 1, 0, 1, 1)
        self.tabWidget = QtWidgets.QTabWidget(self.centralWidget)
        self.tabWidget.setObjectName("tabWidget")
        self.get_task = QtWidgets.QWidget()
        self.get_task.setObjectName("get_task")
        self.connect_button = QtWidgets.QPushButton(self.get_task)
        self.connect_button.setGeometry(QtCore.QRect(10, 10, 101, 29))
        self.connect_button.setObjectName("connect_button")
        self.label = QtWidgets.QLabel(self.get_task)
        self.label.setEnabled(False)
        self.label.setGeometry(QtCore.QRect(260, 10, 110, 30))
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.get_task)
        self.label_2.setEnabled(False)
        self.label_2.setGeometry(QtCore.QRect(370, 10, 66, 30))
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.get_task)
        self.label_3.setEnabled(False)
        self.label_3.setGeometry(QtCore.QRect(120, 10, 81, 30))
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.get_task)
        self.label_4.setEnabled(False)
        self.label_4.setGeometry(QtCore.QRect(200, 10, 31, 30))
        self.label_4.setObjectName("label_4")
        self.groupBox = QtWidgets.QGroupBox(self.get_task)
        self.groupBox.setEnabled(False)
        self.groupBox.setGeometry(QtCore.QRect(10, 40, 471, 201))
        self.groupBox.setAccessibleName("")
        self.groupBox.setAutoFillBackground(False)
        self.groupBox.setFlat(False)
        self.groupBox.setCheckable(False)
        self.groupBox.setObjectName("groupBox")
        self.textBrowser = QtWidgets.QTextBrowser(self.groupBox)
        self.textBrowser.setGeometry(QtCore.QRect(100, 110, 361, 81))
        self.textBrowser.setObjectName("textBrowser")
        self.widget = QtWidgets.QWidget(self.groupBox)
        self.widget.setGeometry(QtCore.QRect(10, 30, 451, 32))
        self.widget.setObjectName("widget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.widget)
        self.horizontalLayout.setContentsMargins(11, 11, 11, 11)
        self.horizontalLayout.setSpacing(6)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.label_5 = QtWidgets.QLabel(self.widget)
        self.label_5.setObjectName("label_5")
        self.horizontalLayout.addWidget(self.label_5)
        self.number_of_tasks = QtWidgets.QSpinBox(self.widget)
        self.number_of_tasks.setEnabled(False)
        self.number_of_tasks.setWrapping(False)
        self.number_of_tasks.setFrame(True)
        self.number_of_tasks.setKeyboardTracking(False)
        self.number_of_tasks.setProperty("showGroupSeparator", False)
        self.number_of_tasks.setMinimum(1)
        self.number_of_tasks.setMaximum(20)
        self.number_of_tasks.setDisplayIntegerBase(10)
        self.number_of_tasks.setObjectName("number_of_tasks")
        self.horizontalLayout.addWidget(self.number_of_tasks)
        spacerItem1 = QtWidgets.QSpacerItem(40, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout.addItem(spacerItem1)
        self.widget1 = QtWidgets.QWidget(self.groupBox)
        self.widget1.setGeometry(QtCore.QRect(10, 70, 436, 31))
        self.widget1.setObjectName("widget1")
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout(self.widget1)
        self.horizontalLayout_2.setContentsMargins(11, 11, 11, 11)
        self.horizontalLayout_2.setSpacing(6)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.run_button = QtWidgets.QPushButton(self.widget1)
        self.run_button.setObjectName("run_button")
        self.horizontalLayout_2.addWidget(self.run_button)
        spacerItem2 = QtWidgets.QSpacerItem(348, 20, QtWidgets.QSizePolicy.Expanding, QtWidgets.QSizePolicy.Minimum)
        self.horizontalLayout_2.addItem(spacerItem2)
        self.tabWidget.addTab(self.get_task, "")
        self.submit_task = QtWidgets.QWidget()
        self.submit_task.setObjectName("submit_task")
        self.tabWidget.addTab(self.submit_task, "")
        self.gridLayout.addWidget(self.tabWidget, 0, 0, 1, 2)
        MainWindow.setCentralWidget(self.centralWidget)
        self.menuBar = QtWidgets.QMenuBar(MainWindow)
        self.menuBar.setGeometry(QtCore.QRect(0, 0, 508, 26))
        self.menuBar.setObjectName("menuBar")
        self.menuFile = QtWidgets.QMenu(self.menuBar)
        self.menuFile.setObjectName("menuFile")
        self.menuEdit = QtWidgets.QMenu(self.menuBar)
        self.menuEdit.setObjectName("menuEdit")
        self.menuAbout = QtWidgets.QMenu(self.menuBar)
        self.menuAbout.setObjectName("menuAbout")
        MainWindow.setMenuBar(self.menuBar)
        self.statusBar = QtWidgets.QStatusBar(MainWindow)
        self.statusBar.setSizeGripEnabled(True)
        self.statusBar.setObjectName("statusBar")
        MainWindow.setStatusBar(self.statusBar)
        self.actionSettings = QtWidgets.QAction(MainWindow)
        self.actionSettings.setCheckable(False)
        self.actionSettings.setEnabled(True)
        self.actionSettings.setObjectName("actionSettings")
        self.actionQuit = QtWidgets.QAction(MainWindow)
        self.actionQuit.setObjectName("actionQuit")
        self.actionAbout = QtWidgets.QAction(MainWindow)
        self.actionAbout.setObjectName("actionAbout")
        self.actionLicense_Information = QtWidgets.QAction(MainWindow)
        self.actionLicense_Information.setObjectName("actionLicense_Information")
        self.menuFile.addAction(self.actionQuit)
        self.menuEdit.addAction(self.actionSettings)
        self.menuAbout.addAction(self.actionAbout)
        self.menuAbout.addAction(self.actionLicense_Information)
        self.menuBar.addAction(self.menuFile.menuAction())
        self.menuBar.addAction(self.menuEdit.menuAction())
        self.menuBar.addAction(self.menuAbout.menuAction())

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.quit_button.setText(_translate("MainWindow", "Quit"))
        self.connect_button.setText(_translate("MainWindow", "Connect"))
        self.label.setText(_translate("MainWindow", "Available tasks:"))
        self.label_2.setText(_translate("MainWindow", "N/A"))
        self.label_3.setText(_translate("MainWindow", "Connected:"))
        self.label_4.setText(_translate("MainWindow", "No."))
        self.groupBox.setTitle(_translate("MainWindow", "Run settings:"))
        self.label_5.setText(_translate("MainWindow", "Number of tasks:"))
        self.run_button.setText(_translate("MainWindow", "Run!"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.get_task), _translate("MainWindow", "Get Tasks"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.submit_task), _translate("MainWindow", "Submit Tasks"))
        self.menuFile.setTitle(_translate("MainWindow", "File"))
        self.menuEdit.setTitle(_translate("MainWindow", "Edit"))
        self.menuAbout.setTitle(_translate("MainWindow", "About"))
        self.actionSettings.setText(_translate("MainWindow", "Settings"))
        self.actionQuit.setText(_translate("MainWindow", "Quit"))
        self.actionAbout.setText(_translate("MainWindow", "About ROctopus"))
        self.actionLicense_Information.setText(_translate("MainWindow", "License Information"))
